cmd/snap/cmd_set.go

```diff
@@ -69,7 +70,26 @@ func init() {
 	})
 }
 
+func fixupRefreshHold(snap, key, value string) string {
```

Comment:
I wouldn't use "fixup" here, maybe something along the lines of "convert" or "compute" or "calculate" or somesuch